cmd/gaia/cmd/gaiacli/main.go

```diff
@@ -80,6 +80,7 @@ func main() {
 		stakecmd.GetCmdQueryUnbondingDelegation(storeStake, cdc),
 		stakecmd.GetCmdQueryUnbondingDelegations(storeStake, cdc),
 		stakecmd.GetCmdQueryValidator(storeStake, cdc),
+		stakecmd.GetCmdQueryValidatorDelegations("stake", cdc),
```

Comment:
```suggestion
		stakecmd.GetCmdQueryValidatorDelegations(storeStake, cdc),
```

Reply:
Should we add this to baseapp ?

Reply:
Technically this isn't the store name, it's actually the query router path.  Just so happens in this case to be the same.

Reply:
Ohhh I see.

Reply:
The LCD endpoints that use queriers, should not use the store name and that should be fixed.  Some of the endpoints may still be left over that do direct store queries, and in that case we pass in the storeName